ppr-ui/src/components/parties/party/EditParty.vue

```diff
@@ -369,16 +371,18 @@ export default defineComponent({
           return
         }
         if (currentSecuredParty.value.businessName && partyType.value === SecuredPartyTypes.BUSINESS) {
-          // go to the service and see if there are similar secured parties
-          const response: [SearchPartyIF] = await partyCodeSearch(
-            currentSecuredParty.value.businessName, false
-          )
-          // check if any results
-          if (response?.length > 0) {
-            // show secured party selection popup
-            showDialog()
-            localState.dialogResults = response?.slice(0, 50)
-            return
+          if (!isEqual(currentSecuredParty, originalSecuredParty)) {
```

Comment:
Does lodash `isEqual` provide a deep comparison on any/all nested objects properties. 
It does state `objects` as supported but perhaps verify it will handle nested properties, ie `personName`, `contact` and so on

Reply:
Will verify, thanks!

Reply:
@cameron-eyds docs say it performs a deep comparison, also did some tests to make sure